doc/examples/data/plot_general.py

```diff
@@ -45,3 +45,28 @@
         plt.imshow(image)
 
 plt.show()
+
+
+"""
+Thumbnail image for the gallery
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+from skimage import data
```

Comment:
Can you move the numpy import after the matplotlib ones above (the other two here are duplicate imports)

```suggestion
```